pkg/controller/nodeipam/ipam/multi_cidr_range_allocator.go

```diff
@@ -262,13 +280,77 @@ func (r *multiCIDRRangeAllocator) Run(stopCh <-chan struct{}) {
 			defer raWaitGroup.Done()
 			r.worker(stopCh)
```

Comment:
we should do a followup and move the node worker to the same pattern, it seems we just have to add a new queue for that ...

Reply:
Ack